.circleci/config.yml

```diff
@@ -53,9 +53,9 @@ jobs:
       - restore_cache:
           keys:
             - repo-{{ .Environment.CIRCLE_SHA1 }}
-      # HACK(albrow): we need to sleep 15 seconds to ensure the devnet is
+      # HACK(albrow): we need to sleep 10 seconds to ensure the devnet is
```

Comment:
We can reduce this wait time now because we are no longer waiting for transactions to be mined in __run.sh__.